scrapy/crawler.py

```diff
@@ -41,6 +41,7 @@ def __init__(self, spidercls, settings=None):
         self.stats = load_object(self.settings['STATS_CLASS'])(self)
 
         handler = LogCounterHandler(self, level=self.settings.get('LOG_LEVEL'))
+        handler.crawler.stats.inc_value('log_count/INFO')
```

Comment:
Instead of incrementing the value directly, maybe it would be feasible to add the handler earlier or move logging after the handler? E.g. at least here we could issue the ``logger.info("Overridden settings: %(settings)r", {'settings': d})`` after adding the log handler, what do you think?

Reply:
I thought of doing this, but when tests are failed, I left it.
I will do it @lopuhin

Reply:
Out of 5, we can handle 1 through this modification

Reply:
@maramsumanth I think @lopuhin's suggestion is not applied yet: logger.info call happens before handler is installed. Manual inc_value here doesn't look good to me.

Reply:
Done @kmike